src/commands/plugins/install.ts

```diff
@@ -95,6 +97,13 @@ e.g. If you have a core plugin that has a 'hello' command, installing a user-ins
 
     const [splitName, tag = 'latest'] = input.split('@')
     const name = await this.plugins.maybeUnfriendlyName(splitName)
+    validateNpmPkgName(name)
     return {name, tag, type: 'npm'}
   }
 }
+
+function validateNpmPkgName(name:string): void {
+  if (!validate(name).validForNewPackages) {
```

Comment:
I'm looking at the npm docs https://github.com/npm/validate-npm-package-name#legacy-names

I can't tell how far in time you'd have to go to hit an "old name"

What do you think the chances are that someone using `oclif` has an npm package that violates the new rules (ex: has a Capital letter in it)?

For safety, I'd consider doing this
1. make this a major version bump
2. since we're doing a major anyway, this is a good time to bump `pjson.engines` to 16+

then we can bump this in our CLIs and most oclif CLIs could, too, unless they happen to need some old-npm -named-plugin?

Reply:
I found this issue from 2013, which is before oclif was released:
https://github.com/npm/npm/issues/3914

Not sure about any other legacy rule, let's do a new major 👍🏼